frontend/components/hint.jsx

```diff
@@ -75,16 +75,15 @@ class Hint extends React.Component {
     const { show } = this.state;
 
     let hintText;
-    switch (hint) {
+    switch (parseInt(hint)) {
     case 1: hintText = "Each table has a unique set of letters. It seems odd that there isn't a table 26 though."; break;
```

Comment:
i dont think there's any reason to switch on the numbers themselves, lets just change these to strings rather than parsing

Reply:
works for me.